source/includes/azure/_network_security_groups.md

```diff
@@ -56,3 +56,26 @@ Attributes | &nbsp;
 `numberOutbound`<br/>*int* | The number of security rules (default and custom) which are outbound
 `numberAssociatedSubnets`<br/>*int* | The number of subnets associated with this network security group
 `numberAssociatedNetworkInterfaces`<br/>*int* | The number of network interfaces associated with this network security group
+
+<!-------------------- CREATE A NETWORK SECURITY GROUP -------------------->
```

Comment:
Do we need to add the region fetcher as part of the documentation as well?

Reply:
d9fb52a10eb96d7e7c62dc0ec7158c0bfd9560b3